server/responses/types.ts

```diff
@@ -43,3 +43,27 @@ export interface AdventureResponse {
 export type AdventuresResponse = {
   adventures: AdventureResponse[];
 }
+
+export interface InitiativeOrderItemResponse {
+  entityId: string;
+  entityType: 'creature' | 'player';
```

Comment:
I think these entityTypes could be an enum since I think they will be referred to more than once, like:

```
export enum EntityType {
    CREATURE = "creature",
    PLAYER = "player"
}
```
so you can refer to it as `EntityType.CREATURE | EntityType.PLAYER`